### ISSUE TITLE
Allow cortextool to delete a whole namespace

### ISSUE THREAD

**Issue by gotjosh** [grafana/cortex-tools#68]
Right now when you try to delete, it only allows a group specification.

**Comment by s-h-a-d-o-w** [comment]
Yup, would be great. Plus, it would be handy to also be able to delete all namespaces with one command.
